config/application.rb

```diff
@@ -40,7 +40,7 @@ class Application < Rails::Application
     config.action_mailer.default_url_options = { host: ENV['MADELINE_HOSTNAME'] }
 
     # Send email on errors
-    unless Rails.env.test?
+    unless Rails.env.test? || Rails.env.development?
```

Comment:
Some dev mode configs can get meaningful error emails in this scenario, but if we want to disable this, we should just do if staging/production rather than unless test/development, resolving unless with compound boolean expressions tends to get confusing.

Reply:
will remove this change